src/rpc/rawtransaction.cpp

```diff
@@ -158,10 +158,12 @@ UniValue getrawtransaction(const JSONRPCRequest& request)
 
     CTransactionRef tx;
     uint256 hashBlock;
-    if (!GetTransaction(hash, tx, Params().GetConsensus(), hashBlock, true))
-        throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, std::string(fTxIndex ? "No such mempool or blockchain transaction"
-            : "No such mempool transaction. Use -txindex to enable blockchain transaction queries") +
-            ". Use gettransaction for wallet transactions.");
+    if (!GetTransaction(hash, tx, Params().GetConsensus(), hashBlock, true)) {
+        std::string msg = std::string(fTxIndex ? "No such mempool or blockchain transaction." :
```

Comment:
nit: missing a space after end of sentence?

Reply:
Not sure I follow?

The row you're commenting on is the message when `fTxIndex` is true, i.e. `-txindex` is specified:
```
$./src/bitcoin-cli getrawtransaction 5644aC275cfe20ca22f7de6bdd8c5313317b5937ae4bcefb6f109ca0dfdc346db
error code: -5
error message:
No such mempool or blockchain transaction.
```

There is no space at the end of the sentence because that's the end of the message; the longer message on L163:164 for when `-txindex` is not set does have a space after the first sentence:
```
$ ./src/bitcoin-cli getrawtransaction 5644a275cfe20ca22f7de6bdd8c5313317b5937ae4bcefb6f109ca0dfdc346db
error code: -5
error message:
No such mempool transaction. Use -txindex to enable blockchain transaction queries. Use gettransaction for wallet transactions, if wallet is enabled.
```

Reply:
Sorry, you are right.